src/amo/pages/Home/index.js

```diff
@@ -255,147 +255,156 @@ export class HomeBase extends React.Component {
         {_config.get('enableFeatureHeroRecommendation') &&
         clientApp !== CLIENT_APP_ANDROID &&
         heroShelves ? (
-          <div className="Home-HeroShelves">
-            <HeroRecommendation shelfData={heroShelves.primary} />
-            <SecondaryHero shelfData={heroShelves.secondary} />
-          </div>
-        ) : null}
-
-        {!_config.get('enableFeatureHeroRecommendation') ||
-        clientApp === CLIENT_APP_ANDROID ? (
-          <HomeHeroGuides />
+          <HeroRecommendation shelfData={heroShelves.primary} />
         ) : null}
 
-        <LandingAddonsCard
```

Comment:
Why is there so much diff churn in here? I had trouble making sense of the change but the screens look OK. I'm just wondering.

Reply:
I don't really know why the diff is so ugly. The actual changes are minimal, but I guess because the markup was refactored things were moved around just enough to totally confuse the diff renderer. I'm not sure if I can do anything to the code to avoid this. 🤷‍♂